src/features/ListAllMovies/ui/MovieDetails.tsx

```diff
@@ -40,6 +44,18 @@ export const MovieDetails: FC<MovieDetailsProps> = ({
                         <p>{transformMinutesToHours(duration)}</p>
                     </div>
                     <p className="w-3/4 text-sm">{description}</p>
+                    <Link to={`/movie/${id}`}>
```

Comment:
let's use constant for path + u can create util method for creating such dynamic paths